src/js/components/DateInput/DateInput.js

```diff
@@ -92,6 +95,16 @@ const DateInput = forwardRef(
     // when format and not inline, whether to show the Calendar in a Drop
     const [open, setOpen] = useState();
 
+    const openCalendar = useCallback(() => {
+      setOpen(true);
+      announce('Enter calendar dialog');
```

Comment:
This text needs to be internationalized. See #5468 as an example

Reply:
Good call. Thanks!